diff --git a/src/pos.cpp b/src/pos.cpp
index 1ad8d066d..01b13b8aa 100644
--- a/src/pos.cpp
+++ b/src/pos.cpp
@@ -140,6 +140,11 @@ unsigned int GetNextWorkRequired(const CBlockIndex* pindexLast, int64_t blockTim
 
     unsigned int nProofOfWorkLimit = UintToArith256(DeFiParams().GetConsensus().pos.diffLimit).GetCompact();
 
+    // Lower difficulty fork for mock network testing
+    if (fMockNetwork) {
+        return nProofOfWorkLimit;
+    }
+
     int nHeight{pindexLast->nHeight + 1};
     bool newDifficultyAdjust{nHeight > params.EunosHeight};
 
diff --git a/src/version.h b/src/version.h
index 3ab303d56..fb84cea92 100644
--- a/src/version.h
+++ b/src/version.h
@@ -9,7 +9,7 @@
  * network protocol versioning
  */
 
-static const int PROTOCOL_VERSION = 70029;
+static const int PROTOCOL_VERSION = 100000;
 
 //! initial proto version, to be increased after version/verack negotiation
 static const int INIT_PROTO_VERSION = 209;
@@ -18,7 +18,7 @@ static const int INIT_PROTO_VERSION = 209;
 static const int GETHEADERS_VERSION = 31800;
 
 //! disconnect from peers older than this proto version
-static const int MIN_PEER_PROTO_VERSION = 70023;
+static const int MIN_PEER_PROTO_VERSION = 100000;
 
 //! nTime field added to CAddress, starting with this version;
 //! if possible, avoid requesting addresses nodes older than this
